common/index.html

```diff
@@ -20,6 +20,30 @@
     <meta name="twitter:creator" content="<%= htmlWebpackPlugin.options.twitter.creator %>" />
     <meta name="google-site-verification" content="dRWkvANAUNAhNyMnTyc7M7S3lnucotMY8j8R-gsZhbo" />
     <link rel="manifest" href="/manifest.json" />
+
+    <script> (function (d, s) {
+      var js = d.createElement(s),
+        sc = d.getElementsByTagName(s)[0];
+      js.src = "https://paywall.unlock-protocol.com/static/unlock.1.0.min.js";
```

Comment:
Possible to load this via an npm module instead of loading via `paywall.unlock-protocol.com`?

Reply:
We did not think of this but I do think that this could be possible. I'll check with the rest of our team and update the PR accordingly!